eng/test-configuration.json

```diff
@@ -7,6 +7,9 @@
     {"testName": {"contains": "MaxRequestBufferSizeTests.LargeUpload" }},
     {"testName": {"contains": "ReaderThrowsResetExceptionOnInvalidBody" }},
     {"testName": {"contains": "AttributeRouting_RouteNameTokenReplace_Reachable" }},
+    {"testName": {"contains": "Http3RequestTests"}},
+    {"testName": {"contains": "Http3TlsTests"}},
+    {"testName": {"contains": "StreamPool_StreamAbortedOnClientAndServer_NotPooled"}},
     {"testName": {"contains": "ServerCertificateSelector_Invoked"}},
```

Comment:
This test is in `Http3TlsTests` but I'm not removing it because it may be flaky for more specific reasons e.g. we might remove the class from here and immediately need to add the specific test back